test/unit/adapters/http.js

```diff
@@ -244,7 +244,8 @@ describe('supports http with nodejs', function () {
       var success = false, failure = false, error;
 
       axios.get('http://localhost:4444/', {
-        maxContentLength: 2000
+        maxContentLength: 2000,
+        maxBodyLength: 2000
```

Comment:
Please write a new test for `maxBodyLength`. Here is testing `maxContentLength` correctly.

Reply:
Test implemented 👌